src/app/pages/observations/observation-detail.component.html

```diff
@@ -328,45 +328,54 @@ <h3>{{'List of evidence' | translate}}</h3>
                 </div>
               </li>
             </ul>
-            <p *ngIf="!documents.length">{{'No evidence' | translate}}</p>
+            <p *ngIf="!documents.length && !isEvidenceTypeOnReport(evidenceType)">{{'No evidence' | translate}}</p>
+            <p *ngIf="isEvidenceTypeOnReport(evidenceType)">{{'observation.evidence.useReport' | translate}}</p>
           </div>
           <div *ngIf="!isDisabled()" class="small-12 medium-6 columns">
             <!-- 9.2.1 Type -->
-            <div [ngClass]="{ 'form-group': true, '-required': !!evidence.attachment || !documents.length }">
+            <div class="form-group -required">
```

Comment:
Would it be possible to move the type field _above_ the rest of the fields and evidence list?

It would look like this:

<p align="center">
<img width="1034" alt="Screenshot of the changes described above" src="https://user-images.githubusercontent.com/6073968/75150007-3d183c80-56fb-11ea-8a68-51f6b3d3dc22.png">
</p>

The reason for the change is that the user would understand better that the evidence type is not per file.

We could also move the “Details” field and hide the rest of the UI when selecting the report option:

<p align="center">
<img width="1036" alt="Screenshot of the changes described above" src="https://user-images.githubusercontent.com/6073968/75150211-ce87ae80-56fb-11ea-8280-08b1c3757c24.png">
</p>